packages/cli/package.json

```diff
@@ -28,7 +28,7 @@
     "test": "export TZ=UTC && jest --ci --silent"
   },
   "dependencies": {
-    "@celo/walletkit": "^0.0.13",
+    "@celo/walletkit": "^0.0.14",
```

Comment:
Given the changes, would it be better to pin to a specific version (remove the ^)?